src/rpc/blockchain.cpp

```diff
@@ -1611,6 +1611,7 @@ static UniValue getchaintxstats(const JSONRPCRequest& request)
     }
 
     const CBlockIndex* pindexPast = pindex->GetAncestor(pindex->nHeight - blockcount);
+    assert(pindexPast != nullptr);
```

Comment:
We don't use assert in rpc code anymore. please use the `CHECK_NONFATAL` to return a bug report to the user instead of crashing the whole server

Reply:
Updated